diff --git a/CMakeLists.txt b/CMakeLists.txt
index 36b8e66..f2cef2d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -431,6 +431,9 @@ if(VLLM_GPU_LANG STREQUAL "CUDA" OR VLLM_GPU_LANG STREQUAL "HIP")
           GIT_SHALLOW TRUE
       )
     endif()
+    if(DEFINED ENV{CUTLASS_NVCC_ARCHS})
+      set(CUTLASS_NVCC_ARCHS "$ENV{CUTLASS_NVCC_ARCHS}" CACHE STRING "CUDA architectures for CUTLASS")
+    endif()
     FetchContent_MakeAvailable(cutlass)
 
     list(APPEND VLLM_STABLE_EXT_SRC
