diff --git i/cmake/cpu_extension.cmake w/cmake/cpu_extension.cmake
index 0000000..0000001 100644
--- i/cmake/cpu_extension.cmake
+++ w/cmake/cpu_extension.cmake
@@ -226,6 +226,8 @@ if (ENABLE_X86_ISA OR (ASIMD_FOUND AND NOT APPLE_SILICON_FOUND) OR POWER9_FOUND
     if(ASIMD_FOUND)
         # Set number of parallel build processes
         include(ProcessorCount)
+        # Allow disabling ACL (e.g. in conda-forge where it is not packaged)
+        if(NOT DEFINED ENV{VLLM_SKIP_ACL})
         ProcessorCount(NPROC)
         if(NOT NPROC)
             set(NPROC 4)
@@ -282,6 +284,9 @@ if (ENABLE_X86_ISA OR (ASIMD_FOUND AND NOT APPLE_SILICON_FOUND) OR POWER9_FOUND
         # VLLM/oneDNN settings for ACL
         set(ONEDNN_AARCH64_USE_ACL ON CACHE BOOL "" FORCE)
         add_compile_definitions(VLLM_USE_ACL)
+        else()
+            message(STATUS "Skipping ACL (VLLM_SKIP_ACL is set)")
+        endif()  # VLLM_SKIP_ACL
     endif()

     set(FETCHCONTENT_SOURCE_DIR_ONEDNN "$ENV{FETCHCONTENT_SOURCE_DIR_ONEDNN}" CACHE PATH "Path to a local oneDNN source directory.")
